enterprise/enterprise_hooks/lakera_ai.py

```diff
@@ -131,3 +126,11 @@ async def async_moderation_hook(  ### 👈 KEY CHANGE ###
                 )
 
         pass
+
+    def format_messages(messages: list) -> str:
```

Comment:
isn't this a duplicate of `get_formatted_prompt` from utils.py 

https://github.com/BerriAI/litellm/blob/88eb25da5c536ef3f95759acfa1db84a7b260c86/litellm/utils.py#L4133